modules/virt-defining-watchdog-device-vm.adoc

```diff
@@ -23,26 +23,28 @@ apiVersion: kubevirt.io/v1
 kind: VirtualMachine
 metadata:
   labels:
-    kubevirt.io/vm: vm2-rhel84-watchdog
+    kubevirt.io/vm: <vm-label> <1>
   name: <vm-name>
 spec:
   runStrategy: Halted
   template:
     metadata:
       labels:
-        kubevirt.io/vm: vm2-rhel84-watchdog
+        kubevirt.io/vm: <vm-label> <1>
     spec:
       domain:
         devices:
           watchdog:
             name: <watchdog>
-            i6300esb:
-              action: "poweroff" <1>
+            <watchdog-device>: <2>
+              action: "poweroff" <3>
 # ...
 ----
-<1> Specify `poweroff`, `reset`, or `shutdown`.
+<1> The label used to identify the VM. For `x86` specify `vm2-rhel84-watchdog`. For `s390x` specify `vm-alpine`.
+<2> The watchdog device to use. For `x86` specify `i6300esb`. For `s390x` specify `diag288`.
+<3> Specify `poweroff`, `reset`, or `shutdown`.
```

Comment:
Should we mention that shutdown is not recommended as stated here: 

https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/6/html/virtualization_administration_guide/section-libvirt-dom-xml-watchdog